src/ucp/api/ucp.h

```diff
@@ -724,6 +740,29 @@ typedef struct ucp_worker_attr {
      * Thread safe level of the worker.
      */
     ucs_thread_mode_t     thread_mode;
+
+    /**
+     * Worker address, which can be passed to remote instances of the UCP library
+     * in order to connect to this worker. The memory for the address handle is
+     * allocated by @ref ucp_worker_query "ucp_worker_query()" routine, and
+     * must be released by using  @ref ucp_worker_release_address
+     * "ucp_worker_release_address()" routine.
+     */
+    ucp_address_t         *address;
+
+    /**
+     * Size of worker address in bytes.
+     */
+    size_t                address_length;
+
+    /**
+     * Flags indicating requested details of the worker address.
+     * If UCP_WORKER_ATTR_FIELD_ADDRESS_FLAGS bit is set in the field_mask,
+     * this value should be set as well. Possible flags are specified
+     * in @ref ucp_worker_address_flags_t. If the value is not set or set to 0,
+     * the returned worker adress will contain maximum details.
```

Comment:
this statement is irrelevant, what if we add more address flags which not reduce details, but do something else?